### PR TITLE
God mode UI

### PR DESCRIPTION
**Pull request by jcarlos0305**
https://user-images.githubusercontent.com/6886356/120834149-ee468d00-c562-11eb-9b0b-d94d01779007.mp4

Finished the implementation of the God Mode UI

Ps. Removed unnecessary logs

### CODE REVIEW

**Comment by jcarlos0305** [comment]
Task linked: [CU-hk6cme Invincible mode script](https://app.clickup.com/t/hk6cme)

File: Penteract/Assets/Scripts/AIMovement.cpp [hunk 1]
```diff
@@ -58,7 +58,6 @@ void AIMovement::Update() {
         else if (state == AIState::RUN) {
             animation->SendTrigger("RunDeath");
         }
-        Debug::Log("Death");
```
Comment (magalenyo) [inline Penteract/Assets/Scripts/AIMovement.cpp:61]: :D

File: Penteract/Assets/Scripts/GodModeController.cpp [hunk 2]
```diff
@@ -112,10 +114,26 @@ void GodModeController::OnChildToggle(unsigned int index, bool isChecked) {
 		break;
 	case 5:
 		if (isChecked) {
-			if (invincible) invincible->Enable();
+			if (playerControllerScript) playerControllerScript->SetOverpower(true);
```
Comment (magalenyo) [inline Penteract/Assets/Scripts/GodModeController.cpp:117]: You could just if (ischecked) before the switch so you don't need to repeat it in every case

**Review by magalenyo — APPROVED** [review]

**Review by isaacfdez — APPROVED** [review]

**Review by Yonimevicio — APPROVED** [review]

**Review by Jordi-Gil — APPROVED** [review]
Tested

**Pull request merged**